### PR TITLE
Add SubnetIds ref to ec2 subnet group ids

### PR DESCRIPTION
**Pull request by brucegucode**
Issue #, if available:

https://github.com/aws-controllers-k8s/community/issues/1457 

Description of changes:

Add dbSubnetIds ref to ec2 subnet ids 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: config/controller/kustomization.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/rds-controller
-  newTag: v0.1.0
+  newTag: v0.0.0-non-release-version
```
Comment (jaypipes) [inline config/controller/kustomization.yaml:9]: uh-oh... this shouldn't have happened... @brucegucode what version of code-generator did you use?

**Review by jaypipes — CHANGES_REQUESTED** [review]
not sure what happened to your local code-generator @brucegucode but it's generating the wrong Helm chart and container image versions...

File: config/controller/kustomization.yaml (hunk 1 above)
Comment (A-Hilaly) [reply config/controller/kustomization.yaml:9]: Might also need to run `git fetch --all --tags`

Comment (brucegucode) [reply config/controller/kustomization.yaml:9]: Ah, i see, need to fetch the latest tags

Comment (brucegucode) [reply config/controller/kustomization.yaml]: updated in rev2 with correct tag

File: apis/v1alpha1/ack-generate-metadata.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-09-02T16:27:09Z"
-  build_hash: cdb0d99c7c1d3d00e71623eb56e650960dcef0d5
-  go_version: go1.18.3
-  version: v0.19.3-2-gcdb0d99
-api_directory_checksum: 83bde1f77a49ebd8acd36a1c5e231138b67aa883
+  build_date: "2022-09-06T20:20:20Z"
+  build_hash: 585f06bbd6d4cc1b738acb85901e7a009bf452c7
+  go_version: go1.18.2
+  version: v0.20.0
```
Comment (A-Hilaly) [inline apis/v1alpha1/ack-generate-metadata.yaml:5]: nit: It is not visible in this case, but generating controller using non matching `runtime`/`code-generator` versions "can" cause problems. Ideally this PR should be generated using code-generator `v0.19.3`

Comment (brucegucode) [reply apis/v1alpha1/ack-generate-metadata.yaml:5]: that's good to know!  it seems after it generated use v0.20.0, it's not able to go backward even i set runtime to v0.19.3 in local

Comment (A-Hilaly) [reply apis/v1alpha1/ack-generate-metadata.yaml:5]: To generate code using `code-generator` v0.19.3, you can run `git checkout tags/v0.19.3`  in code-generator repository  (make sure that you have all tags fetched)

Comment (brucegucode) [reply apis/v1alpha1/ack-generate-metadata.yaml]: thanks! updated to use v0.19.3 and regenerated the code

**Comment by A-Hilaly** [comment]
Nice! Thanks @brucegucode !
/approve
/lgtm

**Pull request merged**